tests/puppeteer/pages/BO/BObasePage.js

```diff
@@ -46,6 +46,10 @@ module.exports = class BOBasePage extends CommonPage {
     this.modalDialog = '#confirmation_modal.show .modal-dialog';
     this.modalDialogYesButton = `${this.modalDialog} button.continue`;
     this.modalDialogNoButton = `${this.modalDialog} button.cancel`;
+
+    // Symfony Toolbar
+    this.sfToolbarMainContentDiv = 'div[id*=\'sfToolbarMainContent\']';
```

Comment:
@boubkerbribri @SimonGrn We can disable the SF toolbar from the SF config, in `app/config/config_test.yml` (this requires Symfony is run in TEST environment), would you like that ?

Also it should be disabled in PROD mode so it means your sanity tests run in DEV mode (for PrestaShop/Symfony) ?

Reply:
> @boubkerbribri @SimonGrn We can disable the SF toolbar from the SF config, in `app/config/config_test.yml` (this requires Symfony is run in TEST environment), would you like that ?

We would like that, yes.

> Also it should be disabled in PROD mode so it means your sanity tests run in DEV mode (for PrestaShop/Symfony) ?

If we install Prestashop in PROD mode (for travis tests), unit and integration tests will failed.

Reply:
Unit and integration tests should be run by different Travis builds. And in fact we should run all our tests in TEST environment, not DEV, and not PROD 😄

Reply:
@matks I'm not totally agreed here.
We must simulate a PROD context in a TEST environment. config_test needs to be updated in order to disable the debug toolbar.
But as far as I know, there are only two env available in prestashop right now, Dev and Prod. We do something here to include the "Test" env here: https://github.com/PrestaShop/PrestaShop/pull/14894